src/quartz_mailer/message.cr

```diff
@@ -0,0 +1,98 @@
+require "email"
+
+class Quartz::Message
+  alias Address = EMail::Address
+
+  getter _from : Address?
```

Comment:
I would consider to initialize the Message class with a named tuple or hash which I think will help cleanup the macros and the class itself

```crystal
property params =  { 
  from: String, 
  to: String, 
  cc: String, 
  bcc: String,
  subject: String, 
  address: String | Address, 
  html: String?, 
  text: String? 
}

def get(key : Symbol)
  params[key]
end

def set(key : Symbol, value)
  params[key] = value
end
```

Reply:
I disagree with storing everything in a hash.  I'm ok with adding the initializer though.

Reply:
> the initializer

@drujensen I don't follow, what do you mean?

Reply:
I think @eliasjpr is proposing that the Message is changed to just a hash.  To me, the Message should be a class.  I am ok with adding an initializer to set all the properties from a hash:
```
class Quartz::Message
  def initialize(params : Hash)
    ... #set all the values
  end
end
```
but I think we should maintain different methods/fields to allow someone to inherit/override if they so desire.